guides/common/modules/proc_migrating-project-or-smartproxy-using-backup-restore.adoc

```diff
@@ -21,8 +21,11 @@ Restore does not significantly alter the target system and requires additional c
 For more information, see {AdministeringDocURL}restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a backup] in _{AdministeringDocTitle}_.
 . Restore the {SmartProxyServer} backup.
 For more information, see {AdministeringDocURL}restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a backup] in _{AdministeringDocTitle}_. 
+. Shut down the old {EL}{nbsp}8 {SmartProxyServer} and reassign its IP address to the new {EL}{nbsp}9 system.
 endif::[]
 ifndef::satellite[]
 . Restore the {ProjectServer} or {SmartProxyServer} backups.
 For more information, see {AdministeringDocURL}restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[Restoring {ProjectServer} or {SmartProxyServer} from a backup] in _{AdministeringDocTitle}_.
+. Shut down the old {EL}{nbsp}8 {ProjectServer} or {SmartProxyServer} and reassign its IP address to the new {EL}{nbsp}9 system.
 endif::[]
+Ensure the DNS resolution works correctly before initiating any tasks.
```

Comment:
The user didn't make any changes to their DNS set up so isn't it confusing to include this?